test/test_load_balancer.py

```diff
@@ -122,6 +122,13 @@ def test_session_gc(self):
         session = client.start_session()
         session.start_transaction()
         client.test_session_gc.test.find_one({}, session=session)
+        # Cleanup the transaction left open on the server.
+        session_id = session.session_id
```

Comment:
This variable is only used once, and the name is pretty self-explanatory. Could it be combined with the next to do:
```
        def kill_session():
             self.client.admin.command("killSessions", [session.session_id])
```
Or, even better maybe would be:
```
self.addCleanup(lambda: self.client.admin.command("killSessions", [session.session_id]))
```

Reply:
No because session is intentionally garbage collected in this test.

Reply:
Oh yes it's possible on second thought. Done.